neptune/utils.py

```diff
@@ -266,3 +266,7 @@ def is_ipython():
         return ipython is not None
     except ImportError:
         return False
+
+
+def is_python_2():
```

Comment:
this function can be replaced with `six.PY2`

Reply:
Done